playbooks/k8s-services.yml

```diff
@@ -42,6 +42,10 @@
 - hosts: kube-master
   become: true
   tasks:
+    - name: Install required OpenShift packages to support Docker registry
```

Comment:
I'm not sure this will help without restarting Kubernetes. I believe OpenShift needs to be installed before helm/tiller is started (as-in in the k8s-cluster.yml).